src/content/blog/projects.mdx

```diff
@@ -0,0 +1,147 @@
+
+Unfortunately, things get _worse_ if you break up a single val into multiple vals that import each other. For example, I broke up the [Date Me Directory](https://val.town/stevekrouse/dateme), an alternative dating website I manage, into a handful of vals, one for each page in the site. Whenever I want to make a change without affecting production, I need to manually fork each of those vals one-by-one, and then manually update all the imports in each of them to point to the forked versions. This was miserable.
+
+<figure>
+  <Image src={DateMe} alt="Val Example" />
+  <figcaption>
+    [Date Me Directory](https://val.town/v/stevekrouse/dateme) - an alternative
+    dating website. It became an unwieldy collection of vals with many
+    disorganized forks.
+  </figcaption>
+</figure>
+
+Instead most users either stuff all their code into a single super long val or just made all their changes directly to their main production vals (sometimes breaking it briefly). Or they moved off Val Town entirely. We needed a solution to dramatically increase the scope of what you can build on Val Town.
+
+## Enter Val Town Projects
+
+Val Town Projects are groups of vals, files, and folders that are versioned, branched, forked, and merged collectively. With Projects, you can create feature branches and test changes without affecting production.
+
+As you may know, Val Town deploys your code in 100ms. This is a magical way to code _directly in the cloud_ – [instead of on localhost and deploying to the cloud later](https://dx.tips/the-end-of-localhost). This kind of live feedback loop is incredibly fun. It's almost addicting. It also means that you are building in the exact same environment that you deploy in, cutting out all the complexity of managing multiple environments.
+
+But just because you're deploying to the _cloud_ on every save doesn't mean you want to deploy to _production_ on every save. Val Town Project branches give you the best of both worlds: you can rapidly prototype in the exact same environment that you'll deploy to, but without affecting production. It's like localhost and PR previews had a baby: super fast iteration cycles, paired with immediately hosted URLs you can share.
+
+And the final kicker is scale. Our vision is that once your app is running on Val Town, it's fully deployed and you can rest easy. You don't have to worry about provisioning more or less compute. Our serverless platform will automatically scale your app to handle any number of users. (Our Scale Plan is slated to launch in Q2 2025. For now, Val Town scales to 100k and 1m runs per day on [the Free and Pro plans](https://www.val.town/pricing), respectively.)
+
+## Projects are not git repos
+
+We considered building Projects as git repos, but decided that becoming a git host at scale was too much complexity for the small subset of git's features that we wanted (diffs, branches, merges). We realized that it'd be much simpler to build our own version control system.
```

Comment:
This feels quite punchy. I guess this is technically true, but I think in my head it's more like "we built a small subset of the features of a version control system in order to avoid building or maintaining our own version control system". maybe I am splitting hairs